CodeGeneration/Sources/SyntaxSupport/GenericNodes.swift

```diff
@@ -93,10 +93,12 @@ public let GENERIC_NODES: [Node] = [
         kind: .collection(kind: .attributeList, collectionElementName: "Attribute", defaultsToEmpty: true)
       ),
       Child(
-        name: "eachKeyword",
-        deprecatedName: "each",
-        kind: .token(choices: [.keyword(.each)]),
-        nameForDiagnostics: "parameter pack specifier",
+        name: "specifier",
+        kind: .token(choices: [
+          .keyword(.each),
+          .keyword(.let)
+        ]),
+        nameForDiagnostics: "specifier",
```

Comment:
Could you add `deprecatedName: "eachKeyword"` so we get compatibility accessors for `eachKeyword`?